testsuite/features/secondary/srv_timezone.feature

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+#@sle_minion
+Feature: Correct timezone display 
+#  1) create a user and assign him a timezone different than the server's timezone
+#  2) test that the popups in some scheduling actions appear in users prefered timezone 
+#  3) some scheduler tests based on previous bugs are unavoidable
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Create a new user in a timezone different than server's timezone
+    When I follow the left menu "Users > User List > Active"
+    And I follow "Create User"
+    And I enter "MalaysianUser" as "login"
+    And I enter "MalaysianUser" as "desiredpassword"
+    And I enter "MalaysianUser" as "desiredpasswordConfirm"
+    And I select "Mr." from "prefix"
+    And I enter "Test" as "firstNames"
+    And I enter "User" as "lastName"
+    And I enter "galaxy-noise@suse.de" as "email"
+    And I select "(GMT+0800) Malaysia" from "timezone"
+    And I click on "Create Login"
+
+  Scenario: Add roles for the Malaysian user
+    When I follow the left menu "Users > User List > Active"
+    And I follow "MalaysianUser"
+    And the "role_satellite_admin" checkbox should be disabled
+    And I check "role_org_admin"
+    And I check "role_system_group_admin"
+    And I check "role_channel_admin"
+    And I check "role_activation_key_admin"
+    And I check "role_config_admin"
+    And I click on "Update"
+
+  Scenario: Login as the new Malaysian user
+    Given I am authorized as "MalaysianUser" with password "MalaysianUser"
+    Then I should see a "MalaysianUser" link
+
+#bsc#1195455, a P3 bug not fixed yet, 
+  Scenario: Schedule a remote script in the future and see the correct timezone as a pop up
+    Given I am on the Systems overview page of this "onalmpantis-min-sles15sp3-1.tf.local"
+    When I follow "Remote Command" in the content area
+    And I enter as remote command this script in
+      """
+      #!/bin/bash
+      ls
+      """
+    And I enter "00:00" as "date_timepicker_widget_input"
+    And I click on "Schedule"
+    Then I should see a "12:00:00 AM MYT" text
+    #WORKAROUND If the above line gets red, they probably fixed the bug, then remove AM from the text
```

Comment:
Better write the workaround comment above the line it refers.

Reply:
sure